src/__internal__/utils/logger/logger.test.ts

```diff
@@ -0,0 +1,24 @@
+import Logger from ".";
+
+test("should not output a warning to the console when logging is disabled", () => {
+  Logger.setEnabledState(false);
+  const consoleWarnSpy = jest
+    .spyOn(console, "warn")
+    .mockImplementation(() => {});
+  Logger.deprecate("This is a deprecation message");
+
+  expect(consoleWarnSpy).not.toHaveBeenCalled();
+});
+
+test("should output a warning to the console with a deprecation prefix when logging is enabled", () => {
+  Logger.setEnabledState(true);
+  const consoleWarnSpy = jest
+    .spyOn(console, "warn")
+    .mockImplementation(() => {});
+  Logger.deprecate("This is a deprecation message");
+
+  expect(console.warn).toHaveBeenCalledWith(
```

Comment:
**suggestion**: apologies @tomdavies73, I overlooked in my previous suggestion that this would cause a lint warning. What do you think of amending it to this to get around it?

```suggestion
  expect(consoleWarnSpy).toHaveBeenCalledWith(
```